parts/windows/windowscsehelper.ps1

```diff
@@ -72,6 +72,7 @@ $global:WINDOWS_CSE_ERROR_INSTALL_CREDENTIAL_PROVIDER = 65 # exit code for insta
 $global:WINDOWS_CSE_ERROR_DOWNLOAD_CREDEDNTIAL_PROVIDER=66 # exit code for downloading credential provider failure
 $global:WINDOWS_CSE_ERROR_CREDENTIAL_PROVIDER_CONFIG=67 # exit code for checking credential provider config failure
 $global:WINDOWS_CSE_ERROR_ADJUST_PAGEFILE_SIZE=68
+$global:WINDOWS_CSE_ERROR_LOOKUP_INSTANCE_DATA_TAG=69 # exit code for looking up nodepool/VM tags via IMDS
```

Comment:
You also need to add the error code name in `$global:ErrorCodeNames`

Reply:
I updated the UT in https://github.com/Azure/AgentBaker/pull/4924

Reply:
thanks, missed this